ftui/screens/dashboard_screen.py

```diff
@@ -281,18 +281,26 @@ def update_dashboard_all_open_trades(self):
         client_dict = self.app.client_dict
         client_dfs = self.app.client_dfs
 
+        trading_mode = "spot"
+
         all_open_df = pd.DataFrame()
         for n, cl in client_dict.items():
+            # if any bots are in futures mode, add leverage column
+            tm = cl.get_client_config().get("trading_mode", "spot")
+            if tm != "spot":
+                trading_mode = tm
             if cl.name in client_dfs and "op_data" in client_dfs[cl.name]:
                 data = client_dfs[cl.name]["op_data"].copy()
                 if not data.empty:
                     all_open_df = pd.concat([all_open_df, data])
 
-        row_data = self._render_open_trade_data(all_open_df)
+        row_data = self._render_open_trade_data(
+            all_open_df, trading_mode=trading_mode
+        )
```

Comment:
won't `trading_mode` here be always the mode whichever the latest client has?

so if i have 3 bots - "spot", "futures", "spot" - then "spot" wins as it's latest ... - which kindof seems to defeat the purpose of passing this to the table?

Reply:
ah you're right, i missed the conditional above 👍